docs/product/guidelines/customization.html

```diff
@@ -12,7 +12,7 @@
 </section>
 
 <section class="stacks-section">
-    {% header h2 | Making Customizations %}
+    {% header h2 | Making customizations %}
```

Comment:
This page should have it's `<h3>'s` checked. Looks like the text for these is defined in this file:
`/docs/_data/product/configurations.yml`